Explore: Logging render performance

davkal opened the pull request:
- calculate Explore results only when query transactions change to prevent expensive re-renders
- moved from grid to flexbox
- split up rendering of graph and log data
- render log results in 2 stages

Using something like react-virtualized would render a lot fewer rows, but I like that the user will be able to use the browser search within the results.

davkal committed:
Explore: Logging render performance

- moved from grid to flexbox
- calculate Explore results only when query transactions change to prevent expensive re-renders
- split up rendering of graph and log data
- render log results in 2 stages

peterholmberg reviewed (changes requested):
Looks cool! And performance is way better. Just some small code things.

davkal committed:
Review feedback

- fixed typo
- moved result calculation to explore utils
- use component keys instead of componentWillReceiveProps
- require logs to have and id, to as use as Logs component key
- render delay based on row count

davkal committed:
Fixed typo in function name
